Vagrantfile

```diff
@@ -86,7 +86,7 @@ DEFAULTS = {
   'ip' => '10.10.10.30',
   'memory' => 1536,
   'themes_dir' => '../alaveteli-themes',
-  'os' => 'jessie64',
+  'os' => 'xenial64',
```

Comment:
I'd missed this initially. This change is smuggled in – the commit is about adding Xenial support, not about changing defaults etc.

I've made a PR to update the default at https://github.com/mysociety/alaveteli/pull/5026, so lets drop this line from this PR.